### ISSUE TITLE
USER ISSUE: Skill Points decreasing!

### ISSUE THREAD

**Issue by Felisto** [StrangeLoopGames/EcoIssues#6335]
**Version:** 0.7.2.5 beta

**Steps to Reproduce:**
For some reason instead of going up, my skill points are constantly going down. I tried everything to "reset" the skill point progress like not eating until hungry and unlearning skills but all my saved up skill points slowly disappear until I am at 0 skill points and then it stays that way.
It only happens with one character on one server (other players on that server don't report this problem).

**Expected behavior:**
Number of Skill Points (and the blue Skill Point bar) should be slowly increasing.

**Actual behavior:**
Instead my Skill Points slowly decrease until I have 0 and the progress bar is empty.

**Comment by craig-jong** [comment]
I'm going to need some logs, reproduction steps, or something for this. This sounds pretty bizarre. My initial thought was that your skill point multiplier might be negative, but since it sounds like its on a multiplayer server, I'm not really sure what it could be.

You're sure none of the other players on the server have this issue? Did you do anything specific to cause this? I know its a bit late, but could you screenshot your skillgain tooltip and stuff next time this occurs?

**Comment by Felisto** [comment]
I sadly can't give you much to work with. 
It happened on 7.2.5 beta. At first the skill points gain stopped (no small green + number above the blue skill bar). When I logged in a few hours later I noticed that I had lost all my saved up skill points. I unlearned something to test it and the blue skill bar progress actually slowly went down until I lost a skill point. 
The next day the problem was gone and I had a few new skill points (but fewer than I should have had).

The tooltip was normal. I checked that. So far the problem hasn't happend a second time.
